### ISSUE TITLE
Method overwrite warning

### ISSUE THREAD

**Issue by tkoolen** [JuliaIntervals/IntervalContractors.jl#31]
https://github.com/JuliaIntervals/IntervalContractors.jl/blob/f03cadd14b6a54fb94598df6cc7ad447e5507f77/src/arithmetic.jl#L4

is causing

```julia
WARNING: Method definition ^(IntervalArithmetic.Interval{Float64}, Base.Rational{T} where T<:Integer) in module IntervalArithmetic at /home/twan/.julia/packages/IntervalArithmetic/9q80F/src/intervals/functions.jl:10 overwritten in module IntervalContractors at /home/twan/.julia/packages/IntervalContractors/EI2Tp/src/arithmetic.jl:4.
```

for me with IntervalArithmetic 0.15.2.

**Comment by dpsanders** [comment]
Thanks for the report. Let me remove that.

**Comment by dpsanders** [comment]
In fact this was #27 which was stalled due to a failing test that has now been fixed in #30.

**Comment by tkoolen** [comment]
Great, thanks!
